tests/ssg_test_suite/rule.py

```diff
@@ -343,7 +350,8 @@ def perform_rule_check(options):
     checker.scenarios_profile = options.scenarios_profile
     # check if target is a complete profile ID, if not prepend profile prefix
     if (checker.scenarios_profile is not None and
-            not checker.scenarios_profile.startswith(OSCAP_PROFILE)):
+            not checker.scenarios_profile.startswith(OSCAP_PROFILE) and
+            not checker.scenarios_profile.lower() == "all"):
```

Comment:
Generally, we don't want to prefix anything to virtual profiles. A simple check could be based on brackets being present only in virtual profiles.

Reply:
done